### ISSUE TITLE
For functions with many arguments, show one argument per line in intellisense documentation (or give option to do so)

### ISSUE THREAD

**Issue by ldorigo** [microsoft/pylance-release#1537]
Currently the popup docs for functions that accept many arguments wraps the function signature on multiple lines and is really hard to read, example from pandas read_excel:

![image](https://user-images.githubusercontent.com/8631181/124900849-be672b00-dfe1-11eb-93b4-c2bd2a8f8d80.png)

IMO, it would be much tidier and more readable to have one argument (+ typing info) per line.

**Comment by ldorigo** [comment]
Or at least, maybe write argument names in bold or in a different color? The current tooltip is so hard to parse it's close to useless to me.

**Comment by judej** [comment]
Dupe of #301
